src/material/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -292,12 +335,17 @@ export class _MatTabLinkBase
   inputs: ['disabled', 'disableRipple', 'tabIndex'],
   host: {
     'class': 'mat-tab-link mat-focus-indicator',
-    '[attr.aria-current]': 'active ? "page" : null',
+    '[attr.aria-controls]': '_tabNavBar.panel ? _tabNavBar.panel.id : null',
+    '[attr.aria-current]': '(active && !_tabNavBar.panel) ? "page" : null',
     '[attr.aria-disabled]': 'disabled',
-    '[attr.tabIndex]': 'tabIndex',
+    '[attr.aria-selected]': '_tabNavBar.panel ? (active ? "true" : "false") : null',
+    '[attr.id]': 'id',
+    '[attr.tabIndex]': '_getTabIndex()',
+    '[attr.role]': '_tabNavBar.panel ? "tab" : null',
     '[class.mat-tab-disabled]': 'disabled',
     '[class.mat-tab-label-active]': 'active',
     '(focus)': '_handleFocus()',
+    '(keydown.space)': '_handleSpace()',
```

Comment:
We don't support any more browsers where `keydown.<key>` doesn't work, but for consistency we should continue using `keyCode` for now. We'll clean all of them up eventually.

Reply:
Done.